spec/reek/spec/should_reek_of_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'pathname'
 require_relative '../../spec_helper'
 require_lib 'reek/spec'
+require 'active_support/core_ext/string/strip'
+require 'active_support/core_ext/hash/except'
```

Comment:
I dont follow. Why would i put that into lib and thus make activesupport a runtime dependency?